proto/storage/compliance_operator_v2.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "gogoproto/gogo.proto";
+
+package storage;
+
+
+// Next Tag: 12
+message ComplianceOperatorProfileV2 {
+    string              id              = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    string              profile_id      = 2;
+    string              name            = 3 [(gogoproto.moretags) = 'search:"Compliance Profile Name,hidden,store" sql:"index=category:unique;name:rule_unique_indicator"'];
+    string              version         = 4 [(gogoproto.moretags) = 'search:"Compliance Operator Version,hidden,store" sql:"index=category:unique;name:rule_unique_indicator"'];
+    string              profile_version = 5 [(gogoproto.moretags) = 'search:"Compliance Profile Version,hidden,store"'];
+    string              product_type    = 6 [(gogoproto.moretags) = 'search:"Compliance Operator Profile Product,hidden,store"'];
```

Comment:
1:1 mapping will not hold true in the case of composite profiles. I recommend an array.